std_traffic/pipelines/sumoToAws.sh

```diff
@@ -412,21 +404,19 @@ fi
 if [ $CLEANUP -eq 1 ]
 then
     logit "Cleaning up..."
-    if [ $SUMO_DONE -eq 0 ]
-    then
-        rm $TMP_DIR/${SUMO_MODEL_PREFIX}.${SUMO_OUTPUT_FILE}.xml*
-    fi
 
     # delete the CSV only if everything else was OK
-    if [ $CSV_DONE -eq 0 ] && [ $S3_DONE -eq 0 ] && [ $DB_DONE -eq 0 ]
+    if [ $S3_DONE -eq 0 ] && [ $DB_DONE -eq 0 ]
     then
         rm $TMP_DIR/${SUMO_MODEL_PREFIX}.${SUMO_OUTPUT_FILE}.csv
     else
         logit "Exiting with error. CSV saved at ${TMP_DIR}/${SUMO_MODEL_PREFIX}.${SUMO_OUTPUT_FILE}.csv"
     fi
 
     if [ $CSV_SPLIT_DONE -eq 0 ]
-    then rm $TMP_DIR/${SUMO_MODEL_PREFIX}.${SUMO_OUTPUT_FILE}.csv.0*
+    then
+        logit "Removing CSV chunks"
+        rm $TMP_DIR/${SUMO_MODEL_PREFIX}.${SUMO_OUTPUT_FILE}.csv.0*
     fi
 fi
 
```

Comment:
OK I guess the answer to my previous comment is: it was temporary :)
I leave it anyway so you can confirm if I understood.